### DISCUSSION TITLE
Can't ignore typescript error in template

### DISCUSSION THREAD

**Discussion by thisismydesign** [nuxt/bridge#535] [Q&A]
### Environment

 Operating System: `Darwin`
- Node Version:     `v16.17.0`
- Nuxt Version:     `2.16.0-27616340.013f051b`
- Nitro Version:    `0.4.24`
- Package Manager:  `yarn@1.22.19`
- Builder:          `webpack`
- User Config:      `server`, `head`, `css`, `env`, `plugins`, `components`, `buildModules`, `modules`, `apollo`, `feed`, `build`, `alias`, `transpileDependencies`, `i18n`, `sitemap`, `auth`, `typescript`, `serverHandlers`, `devServerHandlers`, `bridge`
- Runtime Modules:  `@nuxtjs/axios@5.13.6`, `@nuxtjs/auth@4.9.1`, `@nuxtjs/apollo@4.0.1-rc.5`, `@nuxtjs/feed@2.0.0`, `@nuxtjs/sitemap@2.4.0`, `@nuxtjs/i18n@7.3.0`, `nuxt-rollbar-module@1.1.0`
- Build Modules:    `()`, `@nuxtjs/eslint-module@3.0.2`, `@nuxt/bridge@3.0.0-27700335.6dfc006`

### Reproduction

```jsx
<template>
  <!-- @ts-ignore -->
  <MyComponent :prop="wrongProp" />
</template>
```

### Describe the bug

Can't ignore typescript error in template.

Tried:

`<!-- @ts-expect-error  -->`

`<!-- @ts-ignore -->`

But they have no effect.

Can't really use strict TS this way.

**Comment by danielroe** [comment]
This isn't a Bridge issue. It might be a volar question if that's what you are using.

**Comment by thisismydesign** [comment]
@danielroe I'm using `nuxi typecheck`.. Part of the framework afaict and a pretty common use case

**Comment by danielroe** [comment]
It uses `vue-tsc` under the hood, which is powered by Volar: https://github.com/johnsoncodehk/volar.
